client-registry-jempi/docker-compose.combined.yml

```diff
@@ -3,6 +3,8 @@ version: '3.9'
 services:
   jempi-async-receiver:
     image: jembi/jempi-async-receiver:${JEMPI_AJEMPI_SYNC_RECEIVER_IMAGE_TAG}
+    volumes:
+      - /jempi-csv-data/csv:/app/csv
```

Comment:
I suspect all that is needed is for us to create the folder `/app/csv` in the container and it will work. I'd rather not have this bind mount if possible. I'l try have a look at the dockerfile and see if we could add it there.